content/fr/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -0,0 +1,104 @@
+
+{{< figure src="/images/docs/pod.svg" title="pod diagram" width="50%" >}}
+
+Les Pods fournissent deux types de ressources partagées pour leurs conteneurs : *réseau* and *stockage*.
+
+#### Réseau
+
+Chaque Pod se voit assigner une adresse IP unique. Tous les conteneurs d'un Pod partagent le même namespace réseau, y compris l'adresse IP et les ports réseau. Les conteneurs *à l'intérieur d'un Pod* peuvent communiquer entre eux en utilisant `localhost`. Lorsque les conteneurs dans un Pod communiquent avec des entités *en dehors du Pod*, ils doivent coordonner comment ils utilisent les ressources réseau partagées (comme les ports).
+
+#### Stockage
+
+Un Pod peut spécifier un jeu de *volumes* de stockage partagés. Tous les conteneurs dans le Pod peuvent accéder aux volumes partagés, permettant ces conteneurs de partager des données. Les volumes permettent aussi les données persistantes d'un Pod de survivre au cas où un des conteneurs doit être redémarré. Voir [Volumes](/docs/concepts/storage/volumes/) pour plus d'informations sur la façon dont Kubernetes implémente le stockage partagé dans un Pod.
+
+## Travailler avec des Pods
+
+Vous aurez rarement à créer directement des Pods individuels dans Kubernetes--même des Pods à un seul conteneur. Ceci est dû au fait que les Pods sont conçus comme des entités relativement éphémères et jetables. Lorsqu'un Pod est créé (directement par vous ou indirectement par un Controller), il est programmé pour s'exécuter sur un Nœud dans votre cluster. Le Pod reste sur ce Nœud jusqu'à ce que le process se termine, l'objet pod soit supprimé, le pod soit *expulsé* par manque de ressources, ou le Nœud soit en échec.
+
+{{< note >}}
+Redémarrer un conteneur dans un Pod ne doit pas être confondu avec redémarrer le Pod. Le Pod lui-même ne s'exécute pas, mais est un environnement dans lequel les conteneurs s'exécutent, et persiste jusqu'à ce qu'il soit supprimé.
+{{< /note >}}
+
+Les Pods ne se guérissent pas par eux-mêmes. Si un Pod est programmé sur un Nœud qui échoue, ou si l'opération de programmation elle-même échoue, le Pod est supprimé ; de plus, un Pod ne survivra pas à une expulsion due à un manque de ressources ou une mise en maintenance du Nœud. Kubernetes utilise une abstraction de plus haut niveau, appelée un *Controller*, qui s'occupe de gérer les instances de Pods relativement jetables. Ainsi, même s'il est possible d'utiliser des Pods directement, il est beaucoup plus courant dans Kubernetes de gérer vos Pods en utilisant un Controller. Voir [Pods et Controllers](#pods-and-controllers) pour plus d'informations sur la façon dont Kubernetes utilise des Controllers pour implémenter la mise à l'échelle et la guérison des Pods.
```

Comment:
guérissent => an other verb.

Reply:
@perriea Je ne comprends pas ce que tu veux dire

Reply:
Pour une fois je vais parler en français (oui pas de français dans les review (traçabilité oblige)).
 
Ta phrase `Les Pods ne se guérissent pas par eux-mêmes.` est incorrecte, guérir n'est pas le bon verbe, j'opterai plus pour "recréer".

Reply:
Le phrase originale est : "Pods do not, by themselves, self-heal"
Pourquoi ne pas garder le verbe original ?